src/lib/orm/entities/GuildEntity.ts

```diff
@@ -0,0 +1,505 @@
+import type {
+	CustomCommand, DisabledCommandChannel,
+	NotificationsStreamTwitch, PermissionsNode,
+	RolesAuto, RolesReaction, StickyRole,
+	TriggerAlias, TriggerIncludes, UniqueRoleSet
+} from '@lib/types/settings/GuildSettings';
+import { PREFIX } from '@root/config';
+import { BaseEntity, Column, Entity, PrimaryColumn } from 'typeorm';
+
+@Entity('guilds', { schema: 'public' })
+export class GuildEntity extends BaseEntity {
```

Comment:
Since I honestly can't be asked to write a review for everything, nested props can use https://typeorm.io/#/embedded-entities for a cleaner usage (for example `permissions.users` instead of `permissionsUsers`)

Reply:
Yeah but guild settings aren't ported over for now. This is only there to provide migrations and stuff.